3.6_types.ipynb

```diff
@@ -165,7 +165,7 @@
     "\n",
     "If you make a mistake and mix up `UInt` and `Int` or `Bool` and `Boolean`, the Scala compiler will generally catch it for you.\n",
     "This is because of Scala's static typing.\n",
-    "At compile time, the compiler is able to distinguish between Chisel and Scala types and understand that `if ()` expects a `Boolean` and `when ()` expects a `Bool`.\n"
+    "At compile time, the compiler is able to distinguish between Chisel and Scala types and to understand that `if ()` expects a `Boolean` and `when ()` expects a `Bool`.\n"
```

Comment:
The suggested fix here isn't quite correct. If you really want to be verbose try
```suggestion
    "At compile time, the compiler is able to distinguish between Chisel and Scala types and also able to understand that `if ()` expects a `Boolean` and `when ()` expects a `Bool`.\n"
```